Support/CMake/Toolchain-Android-X86.cmake

```diff
@@ -13,6 +13,8 @@ set(CMAKE_SYSTEM_PROCESSOR X86)
 
 set(CMAKE_C_COMPILER /tmp/aosp-toolchain/x86_64-linux-android-4.9/bin/x86_64-linux-android-gcc)
 set(CMAKE_CXX_COMPILER /tmp/aosp-toolchain/x86_64-linux-android-4.9/bin/x86_64-linux-android-g++)
+set(CMAKE_C_COMPILER_ARG1 -m32)
+set(CMAKE_CXX_COMPILER_ARG1 -m32)
```

Comment:
I didn't even know this was a thing. I'll take a look.

On Thursday, August 18, 2016, Francis Ricci <notifications@github.com>
wrote:

> In Support/CMake/Toolchain-Android-X86.cmake
> <https://github.com/facebook/ds2/pull/428#discussion_r75316801>:
>
> > @@ -13,6 +13,8 @@ set(CMAKE_SYSTEM_PROCESSOR X86)
> >
> >  set(CMAKE_C_COMPILER /tmp/aosp-toolchain/x86_64-linux-android-4.9/bin/x86_64-linux-android-gcc)
> >  set(CMAKE_CXX_COMPILER /tmp/aosp-toolchain/x86_64-linux-android-4.9/bin/x86_64-linux-android-g++)
> > +set(CMAKE_C_COMPILER_ARG1 -m32)
> > +set(CMAKE_CXX_COMPILER_ARG1 -m32)
>
> How about set(triple )? Seems less hacky, would go *before* the
> set(CMAKE_C_COMPILER ) call in the file.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/428/files/c7ab35fae29213c1f75177b353f40d837d088214#r75316801>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADHbBwI0wet9Im7hIZsed-dobE-udksks5qhGqogaJpZM4JnNtz>
> .
>


-- 
Stephane Sezer